portability-gateway/src/main/java/org/dataportabilityproject/gateway/ApiMain.java

```diff
@@ -194,6 +216,14 @@ public ApiServicesModule(
 
     @Override
     protected void configure() {
+      MapBinder<String, AuthServiceExtension> mapBinder =
+          MapBinder.newMapBinder(binder(), String.class, AuthServiceExtension.class);
+
+      for (AuthServiceExtension provider : extensions) {
```

Comment:
nit: if you want, I think this could be done in one line with a functional stream, would follow the functional style of the service loading. something like:

authExtensions.stream().forEach(authExtension -> mapBinder.addBinding(authExtension.getServiceId()).to(authExtension.getClass())

Reply:
Done!